src/app/static/src/app/components/figures/strategise/strategiesTable.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <b-table :items="items" hover selectable select-mode="single" @row-selected="onRowSelected"></b-table>
+  <b-table :items="items" hover selectable select-mode="single" @row-selected="onRowSelected"
+           selected-variant=""></b-table>
```

Comment:
Unsetting `selected-variant` prevents the standard selected-row CSS class being added by BTable, meaning that the relevant styles don't get applied. We can then apply our own.